src/pages/cli/project/troubleshooting.mdx

```diff
@@ -209,6 +209,29 @@ If `amplify push` fails while deploying any of the scenarios below, refer to the
 1.	[Backfill OpenSearch index from DynamoDB table](/cli/graphql/troubleshooting/#deploying-multiple-index-changes-at-once)
 1.	[Index with Multi-Sort Key Fields](/cli/graphql/troubleshooting/#index-with-multiple-sort-key-fields)
 
+## Troubleshooting "amplify pull" failures
+
+### Pull failures with large number of UI components
+<Callout information>
```

Comment:
does this render properly?

Reply:
@vishal-dms and I touched base offline. It looks good!

Reply:
Yes

![amplify-pull-failure](https://github.com/aws-amplify/docs/assets/86995692/138a20d6-30d2-419d-846c-973195e496b5)